triplestore/demo/demo.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2025 Maira Papadopoulou
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import argparse
+from contextlib import suppress
+from typing import Any
+
+from triplestore import Triplestore
+
+GRAPH_IRI = "http://example.org/test"
+
+# Minimal configs per backend (extend with base_url/credentials if needed)
+CONFIGS: dict[str, dict[str, Any]] = {
+    "allegrograph": {"repository": "test2025", "graph": GRAPH_IRI},
+    "blazegraph": {"namespace": "test2025", "graph": GRAPH_IRI},
+    "graphdb": {"repository": "test2025", "graph": GRAPH_IRI},
+    "jena": {"dataset": "test2025", "graph": GRAPH_IRI},
+    "oxigraph": {"graph": GRAPH_IRI},
+}
+
+
+def main() -> None:
+
+    parser = argparse.ArgumentParser(
+        description="Unified triplestore demo showcasing the abstraction layer."
+    )
+    parser.add_argument(
+        "-b",

```

Comment:
Whenever you have a short (one-letter) option, always provide a long one (`--backend`). 

The reverse is obviously not true (pigeonhole principle).